### PR TITLE
Main Class for a Member

### PR DESCRIPTION
**Pull request by stevempope**
Addresses #94 and #109.

The Main class takes in a list of abilities, filters them and starts a member if they have at least one ability.

The member should then be responsible for firing up the required controllers in its constructor.

### CODE REVIEW

File: mcomp_Java/src/member/MemberMain.java [hunk 1]
```diff
@@ -4,19 +4,64 @@
...
+				skills.add(common.datatypes.Ability.VIEWER);
+				break;
+			case "DEST_SETTER":
+				skills.add(common.datatypes.Ability.DEST_SETTER);
+				break;
+			default: break;
+			}
+		}
+		if (skills.isEmpty()) {
+			System.out.println("Sorry, a Pathfinder needs to have at least one ability!");
```
Comment (SamanthaDawsonBanks) [inline mcomp_Java/src/member/MemberMain.java]: Pathfinder?

File: mcomp_Java/src/common/objects/Member.java [hunk 2]
```diff
@@ -145,5 +136,37 @@ public void setHerdID(String newID) {
 		 */
 		herdID = newID;
 	}
+
+
+	@Override
+	public int add(int a, int b) throws RemoteException {
```
Comment (SamanthaDawsonBanks) [inline mcomp_Java/src/common/objects/Member.java:142]: this method is left over from RMI testing but may be a good one for a RMI test "are you alive" call

**Review by SamanthaDawsonBanks — APPROVED** [review]
looks good to me (subject to the two comments)

**Review by RyanShoobert — APPROVED** [review]

File: mcomp_Java/src/member/MemberMain.java (hunk 1 above)
Comment (stevempope) [reply mcomp_Java/src/member/MemberMain.java]: I will change it to robot, do you think that is a better description?

File: mcomp_Java/src/common/objects/Member.java (hunk 2 above)
Comment (stevempope) [reply mcomp_Java/src/common/objects/Member.java:142]: This comes from the RMIable interface so we probably need to have a discussion there at some point.

**Comment by stevempope** [comment]
Added fixes and merging

**Pull request merged**